lib/shared/addon/components/radio-button/component.js

```diff
@@ -14,6 +14,9 @@ export default Component.extend({
   }),
   click() {
     this.set('selection', this.get('value'));
+
+    this.chosen();
```

Comment:
Noop?

Reply:
Yeah - when clicked, it should invoke chosen, which has an empty default which is then over-ridden by uses of the component.

Reply:
lgtm